src/components/Avatar.jsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+function getUserInitials(name) {
+  if (!name) return 'GU';
+  return name
+    .split(' ')
+    .map((part) => part[0])
+    .join('')
+    .toUpperCase()
+    .slice(0, 2);
+}
```

Comment:
move to the end of the file

Reply:
also if not have name return null

Reply:
why this function should be at the end of the file? i think it makes more sense to read the function and then the actual use of it inside the component, right?